library/MedEx/MedEx.php

```diff
@@ -0,0 +1,55 @@
+<?php
+    /**
+     * /library/MedEx/MedEx.php
+     *
+     * This file is the callback service for MedEx to update local DB with new responses in real-time.
+     * It must be accessible remotely to receive data synchronously.
+     *
+     * It is not required if a practice is happy syncing using background services only.
+     *
+     * The advantages using this file are:
+     *  1.  Real time updates of patient responses == synchronous receiver
+     *  2.  Reduced need to run the MedEx_background service
+     *          - MedEx_background syncs DB responses asynchronously, ie only when run (default = every 29 minutes)
+     *          - It consumes resources and may affecting performance of the server if run too often.
+     *              (see MedEx_background.php for configuration examples)
+     *
+     * Uses multiple authentication steps for security:
+     *  - local API_key
+     *  - MedEx username
+     *  - session token
+     *  - MedEx generated token
+     *
+     * @package MedEx
+     * @author MedEx <support@MedExBank.com>
+     * @link    http://www.MedExBank.com
+     * @copyright Copyright (c) 2017 MedEx <support@MedExBank.com>
+     * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+     */
+    
+    $ignoreAuth=true;
+    $_SERVER['HTTP_HOST']   = 'default'; //change for multi-site
+    
+    require_once(dirname(__FILE__)."/../../interface/globals.php");
+    require_once(dirname(__FILE__)."/../patient.inc");
+    require_once(dirname(__FILE__)."/../log.inc");
+    require_once(dirname(__FILE__)."/API.php");
+    require_once(dirname(__FILE__)."/../formatting.inc.php");
```

Comment:
TODO (for me)
remove formatting.inc.php since this is already autoloaded in globals.php